File: Computer-Cluster-Python-master/node.py
class Node:

    def __init__(self, minnestorrelse, prosessorantall):
        self._minnestorrelse = minnestorrelse
        self._prosessorantall = prosessorantall

    def hentProsessorantall(self):
        return self._prosessorantall

    #returnerer True hvis Noden har nok minne
    def nokMinne(self, paakrevdMinne):
        return paakrevdMinne <= self._minnestorrelse

    def printInfo(self):
        print("Minnestørrelse:", self._minnestorrelse)
        print("Prosessorantall:", self._prosessorantall)

File: Computer-Cluster-Java-master/Node.java
public class Node {
  private int minnestrl;
  private int proAntall;

  public Node (int minnestrl, int proAntall) {
    this.minnestrl = minnestrl;
    this.proAntall = proAntall;
  }

  public int hentProant() {
    return proAntall;
  }

  public boolean nokMinne(int paakrevdMinne) {
    if (minnestrl >= paakrevdMinne) {
      return true;
    }
    else {
      return false;
    }
  }
}
